### DISCUSSION TITLE
How do I transfer the catalog?

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#6946] [Q&A] [answered]
Hello everyone! I need some help with transferring a directory containing `extra_css` files from the `docs_dir`. This is causing a lot of problems with my project, as I would like to have only Markdown (md) files in that directory.

This step is not helpful for me:
```
extra_css:
  - ../stylesheets/extra.css
```

Can anyone please help me with this? Thanks in advance!

**Comment by alexvoss** [comment]
You need to put the `extra.css` somewhere that it gets deployed to the site. The MkDocs documentation lists the [static_templates](https://www.mkdocs.org/user-guide/configuration/#static_templates) option for adding static files. Hope that helps.

**Reply by ghost** [reply]
Unfortunately, I did not understand anything at all.

**Reply by kamilkrzyskow** [reply]
You need to be more specific about what didn't help.

How does your current `extra_css` look like in the mkdocs.yml file?

What was the file path to the files before and what is the current path to those files?

What are the problems caused, we guess the CSS doesn't load but perhaps there are other issues you attribute to that change?

**Reply by alexvoss** [reply, marked as answer]
When you put the CSS into `docs/` then it gets deployed to your site, while files outside `docs/` do not. The `extra_css` configuration option does not result in extra files being deployed but just leads to a link being inserted in the HTML that loads the CSS. If you want to keep your CSS, JS or other files outside the `docs/` folder then you need to also get MkDocs to copy them over to your site. 

It turns out that the `static_templates` option is not needed for CSS and JS, so this should be enough:

```
theme:
  name: material
  custom_dir: assets

extra_css:
  - stylesheets/extra.css
 ```

Now, create a folder `assets` in your project root and inside that `stylesheets`, then copy/move your `extra.css` there.

**Comment by ghost** [comment]
Thank you very much, guys! Have a great day!!!
